### ISSUE TITLE
Colorblind Accessibility

### ISSUE THREAD

**Issue by AusIV** [saltstack/halite#190]
On the Job list and Minion list, Halite uses red and green indicators to indicate job success or failure and whether minions are up or down. This is a fairly common convention, but is also inaccessible to red green colorblindness, which affects about 10% of men.

I hovered my mouse over the icons hoping for a clarification, but the title text only tells me that "Green=success, Red=fail". It seems like that would be obvious enough, if I could tell the difference between red and green.

This accessibility issue could be resolved by either changing the icon between successes and failures, or by having the title text indicate success or failure.

**Comment by iggy** [comment]
I wouldn't expect much further development out of halite. SSE has it's own closed source dashboard/UI now. It's doubtful they will put resources into development of something that compete with something they are trying to sell.

There are other alternatives out there. Saltpad, etc. You might want to look into them.

**Comment by patelrahul2040** [comment]
can any one help me to write the SSL file for perforce command line client

**Comment by rallytime** [comment]
This repo is deprecated and archived. Therefore, this issue is now closed. Thank you.
